graylog2-web-interface/src/components/welcome/EntityListItem.tsx

```diff
@@ -42,15 +42,25 @@ type Props = {
 }
 
 const EntityItem = ({ type, title, id }: Props) => {
-  const entityType = entityTypeMap[type] ?? entityTypeMap.unknown;
+  const entityTypeTitle = useMemo(() => {
+    try {
+      return getTitleForEntityType(type);
```

Comment:
We could also add a parameter for `getTitleForEntityType` to define if the function should throw an error when and entity type is unknown. The parameter could be true by default.